website/source/guides/secret-mgmt/versioned-kv.html.md

```diff
@@ -0,0 +1,950 @@
+---
+layout: "guides"
+page_title: "Versioned KV Secret Engine - Guides"
+sidebar_current: "guides-secret-mgmt-versioned-kv"
+description: |-
+  Vault 0.10.0 introduced version 2 of key-value secret engine which supports
+  versioning of your secrets so that you can undo the accidental deletion of
+  secrets, or compare the different versions of the secret.
+---
+
+# Versioned Key/Value Secret Engine
+
+The [Static Secrets](/guides/secret-mgmt/static-secrets.html) guide introduced
+the basics of working with key-value secret engine. **Vault 0.10** introduced [_K/V
+Secrets Engine v2 with Secret
+Versioning_](https://www.hashicorp.com/blog/vault-0-10).  This guide
+demonstrates the new features introduced by the key-value secret engine v2 so
+that you can start leveraging them.
```

Comment:
`This guide demonstrates the new features introduced by the key-value secret engine v2`